contributors/devel/cherry-picks.md

```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The Patch Release Manager
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
+   * The Patch Release Manager for the target release branch is responsible for
+   applying the `cherrypick-approved` label. You can find the release team contacts
+   under the kubernetes/sig release repository - for example, [here is the 1.11 team](https://git.k8s.io/sig-release/releases/release-1.11/release_team.md).
 
 ## Cherry Pick Review
 
 Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
+particular, they may be self-merged by the Patch Release Manager without fanfare,
+in the case the Patch Release Manager knows the cherry pick was already
 requested - this should not be the norm, but it may happen.
```

Comment:
this section is from 2015 and arguably needs a 2018 refactor outlining the current "norm".
i can summarize, if needed.

Reply:
@neolit123 yes, please! I'm not super familiar with the process; I stumbled across it because of a coworker's struggles.